modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/java/com/liferay/frontend/editor/ckeditor/web/internal/editor/configuration/CKEditorConfigContributor.java

```diff
@@ -27,7 +27,10 @@
  * @author Ambrín Chaudhary
  */
 @Component(
-	property = {"editor.name=ckeditor", "editor.name=ckeditor_classic"},
+	property = {
+		"editor.name=ckeditor", "editor.name=ckeditor_classic",
+		"editor.name=ckeditor_extensible"
```

Comment:
You shouldn't need to change `CKEditorConfigContributor`. This property "`editor.name=ckeditor_extensible`" should be added to the new `CommerceCKEditorConfigContributor`. 

(Because `CKEditorConfigContributor` adds the extraPlugin and the button, but you are already doing that in your `CommerceCKEditorConfigContributor`)